app/build.gradle.kts

```diff
@@ -211,3 +211,6 @@ tasks.register("jacocoTestReport", JacocoReport::class) {
         include("outputs/code_coverage/debugAndroidTest/connected/*/coverage.ec")
     })
 }
+dependencies {
+    implementation("androidx.test:core-ktx:1.5.0")
```

Comment:
Put it with the other implementations

Reply:
Thank you for finding this. Sometimes, when you click 'auto import dependency', it for some reason adds it to the bottom of the Gradle file.